packages/@lwc/compiler/src/compiler/options.ts

```diff
@@ -64,10 +64,10 @@ export interface NormalizedDynamicComponentConfig {
     strictSpecifier: boolean;
 }
 
-export interface CompilerOptions {
+export interface TransformationOptions {
```

Comment:
`TransformerOptions` would be the equivalent but that reminds me too much of Optimus Prime. 

What if we have `TransformOptions` and `CompileOptions`? This actually makes more sense because the options do not apply to the compiler itself but to an individual compile operation.